pingcap/tidb
# lightning: Corrected logging variable size to totalSize

### What problem does this PR solve?


Issue Number: close #46084

Problem Summary: Incorrectly logging the size variable instead of totalSize in WriteToTiKV function in br/pkg/lightning/backend/local/local.go

### What is changed and how it works?
Incorrect variable to log size used in WriteToTiKV function.
### Check List

Tests 

- [ X] Manual test (add detailed scripts or steps below)
- [] No code

Manual test steps:
Spin up a cluster.
Install Lightning and import data.
Check logs to verify that the size logged is not 0.

Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Corrected logging variable in WriteToTiKV() from size to totalSize.
```
